lib/helpers/changelog.js

```diff
@@ -0,0 +1,33 @@
+const logger = require('winston');
+const changelog = require('changelog');
+
+module.exports = getChangeLog;
+
+// Get Changelog
+async function getChangeLog(depName, fromVersion, newVersion) {
+  logger.debug(`getChangeLog(${depName}, ${fromVersion}, ${newVersion})`);
+  if (!fromVersion || fromVersion === newVersion) {
+    return '';
+  }
+  const semverString = `>${fromVersion} <=${newVersion}`;
+  logger.debug(`semverString: ${semverString}`);
+  let markdownLog;
```

Comment:
Could you assign an empty string to `markdownLog` here, and save yourself from having line 25.